JitsiConferenceEventManager.js

```diff
@@ -99,13 +111,15 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
 
             Object.keys(chatRoom.connectionTimes).forEach(key => {
                 const value = chatRoom.connectionTimes[key];
+                const eventName = `conference.${_CONNECTION_TIMES_}.${key}`;
```

Comment:
Can you please confirm the accuracy of this event name? I see _CONNECTION_TIMES_ defined as an empty string so would this event name be "conference..foo"?

Reply:
Good catch! Will fix